examples/tv-casting-app/android/App/app/src/main/jni/cpp/support/Converters-JNI.cpp

```diff
@@ -25,6 +25,30 @@ namespace support {
 
 using namespace chip;
 
+jobject convertLongFromCppToJava(jlong value)
```

Comment:
should we consider to move this file into src/lib/support ?

Reply:
Tracking this here: #33029